utils/cLog.h

```diff
@@ -1,48 +1,66 @@
-#ifndef D2TM_CLOG_H_
-#define D2TM_CLOG_H_
+#pragma once
 
-class cLogger {
-
-public:
-	void log(eLogLevel level, eLogComponent component, const std::string& event, const std::string& message);
-	void log(eLogLevel level, eLogComponent component, const std::string& event, const std::string& message, eLogOutcome outcome, int playerId, int houseId);
```

Comment:
this is interesting, so you moved this out of this class. And you put these in the anonymous namespace. I would've expected it to be moved to a `private:` section in the `cLogger` class. What is the rationale not to do this?